File: PRACTICA 2/Linea.py
class Linea:
    def __init__(self, p1, p2):
        self.p1 = p1
        self.p2 = p2

    def __str__(self):
        return f"Linea[{self.p1} -> {self.p2}]"

    def dibujaLinea(self):
        print(f"Dibujando línea desde {self.p1} hasta {self.p2}")

File: PRACTICA 2/Linea.java
class Linea {
    private Punto p1, p2;

    public Linea(Punto p1, Punto p2) {
        this.p1 = p1;
        this.p2 = p2;
    }

    @Override
    public String toString() {
        return "Linea[" + p1 + " -> " + p2 + "]";
    }

    public void dibujaLinea() {
        System.out.println("Dibujando línea desde " + p1 + " hasta " + p2);
    }
}
